### PR TITLE
fixed bug in the corrected MET in the JetID cleaned module

### PR DESCRIPTION
**Pull request by safarzad**
Fix the bug in the HLTMETCleanerUsingJetID.cc.

Remove usePt option "which was wrongly set to false"
to use met.pt() n coherent with standard met definition. 
Modified Cleaned MET collection be massless after correcting for JetID.

### CODE REVIEW

**Comment by Martin-Grunewald** [comment]
please test

(will probably not work)

**Comment by Martin-Grunewald** [comment]
Well, in this case you should add the deletion of the parameter
like is done for similar cases here:
https://github.com/cms-sw/cmssw/blob/CMSSW_8_0_X/HLTrigger/Configuration/python/customizeHLTforCMSSW.py
add this to this PR so that the jenkins tests will pass without error.

As this is a physics change, it should be discussed and signed off on in a TSG meeting.

**Comment by smuzaffar** [comment]
please test

File: HLTrigger/Configuration/python/customizeHLTforCMSSW.py [hunk 1]
```diff
@@ -203,7 +203,12 @@ def customiseFor12062(process):
         module.recordLabel = cms.string('HLT')
     return process
 
+def customiseFor12318(process);
 
+    if hasattr(process, 'hltMetCleanUsingJetID'):
+        delattr(process.hltMetCleanUsingJetID, 'usePt')
```
Comment (Martin-Grunewald) [inline HLTrigger/Configuration/python/customizeHLTforCMSSW.py]: Please protect the delattr by checking: ``if hasattr (process.hltMetCleanUsingJetID,'usePt):```

File: HLTrigger/Configuration/python/customizeHLTforCMSSW.py [hunk 2]
```diff
@@ -217,6 +223,7 @@ def customiseHLTforCMSSW(process, menuType="GRun", fastSim=False):
         process = customiseFor11497(process)
         process = customiseFor12044(process)
         process = customiseFor12062(process)
+        process = customiseFor12318(process)
```
Comment (Martin-Grunewald) [inline HLTrigger/Configuration/python/customizeHLTforCMSSW.py]: This should be under an if CMSSW>8_0....

**Comment by Martin-Grunewald** [comment]
There are now merge conflicts which must be solved!

**Comment by Martin-Grunewald** [comment]
I think you used the wrong version (75X? 76X?) of customizeHLTforCMSSW.py
That file, in 80X, already contains a block with ```if cmsswVersion >= "CMSSW_8_0":```

**Comment by Martin-Grunewald** [comment]
Still merge conflicts...
also, "usePt" should be removed from the plugin's header file.

**Comment by safarzad** [comment]
please do not consider this

**Comment by safarzad** [comment]
To avoid this error I am trying to customize it

**Comment by safarzad** [comment]
Could you please redo the test

**Comment by fwyzard** [comment]
please test

**Comment by Martin-Grunewald** [comment]
Not so useful as there are still merge conflicts according to GitHub.
Please fix them as the PR will not be accepted by David otherwise.

**Comment by safarzad** [comment]
To avid any merging issue with other PR I will wait till 12334 is merged and then redo this PR although do not know what can be wrong.

**Pull request closed without merging**